### ISSUE TITLE
Color scheme in 5.0

### ISSUE THREAD

**Issue by chris-b1** [ipython/ipython#9541]
This is on Windows 10 64-bit, python 3.5, `IPython 5.0.0b1`

Some of the new syntax highlighting, specifically imports, clashes with the [console](https://conemu.github.io/) I'm using.  I tried using `--colors=NoColor`, but this doesn't seem to do anything?  Thanks.

![image](https://cloud.githubusercontent.com/assets/1924092/15806136/7dc79678-2b02-11e6-9c66-f98086b2b545.png)

**Comment by takluyver** [comment]
It uses [pygments styles](http://pygments.org/docs/styles/), and you can pick a different one with e.g.

```
ipython --TerminalInteractiveShell.highlighting_style="paraiso-dark"
```

I'm not sure how to integrate this with the old colors option (which still affects tracebacks and object-inspection output for now), because that uses a completely separate system with a different (and smaller) set of themes.

We tried to pick colours that would work acceptably on both black and white backgrounds, and in most terminals they do. I'm pretty sure I've tested in the normal Windows command prompt, but I didn't test with conemu.

**Comment by chris-b1** [comment]
Awesome, thank you. The default theme does look fine in the standard Windows command prompt, for the record, `'vs'` looks pretty decent in ConEmu.  Feel free to close or leave as a doc issue.

**Comment by Carreau** [comment]
Marking that as documentation.

**Comment by MaximilianR** [comment]
I imagine this isn't the most pressing thing you guys are working on for the 5.0 release, but mentioning here regardless.

The color scheme is _great_ overall. 
But there is some jarring in the differences in the colors between the `[ ]` and the number inside these (i.e. the 6, 7 & 8 below). Unless it's just a personal preferences thing

![image](https://cloud.githubusercontent.com/assets/5635139/15908794/7cb4e97a-2d91-11e6-82d7-a330e0fe4b0c.png)

**Comment by Carreau** [comment]
Yeah, I think this is on purpose, though now you should be able to completely customize it. 

But thanks for the kind words !

**Comment by Carreau** [comment]
Closed by #9655 the `--color=NoColor|Linux|LightBg` has been kind-of restored (see release notes, and previous PR) but is slightly different than 4.0 for now (we need a new Pygments version for that)
